sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionSerializerTest.java

```diff
@@ -30,15 +30,15 @@ public void BeforeEachTest() throws InvalidAlgorithmParameterException, NoSuchAl
         String payload = "This is string to sign";
         var message = payload.getBytes(StandardCharsets.UTF_8);
         var someValue = BigInteger.ONE;
-        var rawTX = RawTransaction.createTransaction(someValue,
+        var rawTransaction = RawTransaction.createTransaction(someValue,
                 someValue, someValue, "0x", someValue, "");
 
         // Create a key pair, create tx signature and create ethereum Transaction
         ECKeyPair pair = Keys.createEcKeyPair();
         var msgSignature = Sign.signMessage(message, pair, true);
         var txSignature = new SignatureSecp256k1(msgSignature);
-        var txProposition = new PublicKeySecp256k1Proposition(Strings.toByteArray("0x" + Keys.getAddress(pair)));
-        ethereumTransaction = new EthereumTransaction(rawTX, txSignature, txProposition);
+        var txProposition = new AddressProposition(BytesUtils.fromHexString(Keys.getAddress(pair)));
+        ethereumTransaction = new EthereumTransaction(rawTransaction);
```

Comment:
let's add one more test, where we check the serialization for signed ethereum tx.